content/blog/embrace-kubernetes-part2/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "It’s Time to Embrace Kubernetes! Really? "
+date: 2020-12-11
+meta_desc: "How to evaluate your choices when choosing Kubernetes as your application platform."
+meta_image: embrace-k8s.png
+authors:
+    - lee-briggs
+tags:
+    - kubernetes
+
+---
+
+In the [first article]({{< relref "/blog/embrace-kubernetes-part1" >}}) in this series,  we gave you some questions to help you and others at your company decide if Kubernetes is right for you. In this post, we’ll give you an example of where Kubernetes can be a good fit.
+
+<!--more-->
+
+## When is Kubernetes a Good Choice?
+
+If you’ve decided your company can make the large investment of time and resources required to adopt Kubernetes, when is it the right choice? A well designed, well implemented Kubernetes platform can certainly deliver value to your customers. It does a fantastic job of reducing complexity from the application developers’ jobs. Just remember that the payoff, although dramatic, will be many months away and, before that, things will be slower than you’re used to.
+
+One example of where Kubernetes can add a lot of value is when the application side of the engineering organization has a low level of maturity, and the operations side has a high degree of maturity. Kubernetes can make the applications developers jobs easier because it abstracts away deployment details. In turn, the operations team must be willing to take on the burden of shipping software and let the application developers concentrate on creating new products and features. This division implies a good relationship between them, and fostering cooperation is always beneficial. If trust and good communication exist, Kubernetes is a great way for engineers to work together.
+
+Here’s one example of how Kubernetes can help. Developers spend large amounts of time writing code that’s resilient to failure. Kubernetes, on the other hand, almost encourages you not to spend time writing fault-tolerant software and let the Kubernetes handle failure. Application developers don’t need to write resiliency into their applications. They can simply let the programs fail quickly and assume that Kubernetes will take care of it.
+
+## Are There Alternatives?
+
+Asking if there are alternatives to Kubernetes makes sense, but it’s unlikely you’ll find a simple solution. Large-scale, complex problems won’t be solved easily. Most open-source alternatives can’t compete against it because Kubernetes is generic and has many building blocks available.
```

Comment:
Is it worth mentioning the cloud-specific alternatives to kubernetes? So ECS on AWS, Container services/web app service on Azure and app engine? Even if there isn't a large amount of detail, at least mentioning them as alternatives

Reply:
I think that alternatives/is Kubernetes the right choice was covered in the first article.